Dmin_autocalculator_config.ini

```diff
@@ -0,0 +1,54 @@
+#works for both, determinD and batchAnalysis. To determin D_min min_detectable_D should be 0, for batch analysis replace it with the D_min you get out of determinD
+[SOFTWARE]
+# choose software, either ThunderSTORM or rapidSTORM
+software = ThunderSTORM
+
+[INPUT_DIRS]
+# path to the coverslips
+dir1 = D:\exampledir
+
+
+[MASK_WORDS]
+#words to filter in your directory, e.g. BACKGROUND
+mask = background
+
+[BACKGROUND]
+#whether to use background correction. If true give directories.
+background = false
+
+[BACKGROUND_DIRS]
+#directories to the background, make sure  they are in the same order as the INPUT_DIRS. Only necessary when background is set to true.
+dir01 = 
```

Comment:
instead of blank space after =, add "" to make clear what the value of the variable will be → dir01 = ""